CHANGELOG.md

```diff
@@ -1,5 +1,75 @@
+- [ENHANCEMENT] rework `percli plugin build` to vendor CUE dependencies (#2695)
+- [ENHANCEMENT] CLI/Migrate: support CR as a migration format (#2682)
+- [ENHANCEMENT] Support conditional queryOptions for panels (#2662)
+- [ENHANCEMENT] Add Authorization and BasicAuth definitions to cue package (#2643)
+- [ENHANCEMENT] Move icons to the overflow menu based on panel size (#2635)
+- [ENHANCEMENT] Unify loading and error handling for all panels, pass data as props to panels (#2549)
+- [ENHANCEMENT] Support min/max TLS version (#2619)
+- [ENHANCEMENT] Disable schema validation by config for plugin in development (#2617)
+- [ENHANCEMENT] Deprecate the schema config (#2614)
+- [ENHANCEMENT] schemas are required for specific plugin kind (#2608)
+- [ENHANCEMENT] Improve panel hover behavior and adjust header styling for better visibility (#2603)
+- [ENHANCEMENT] `percli dac preview`: add current dashboard URL to the output when applicable (#2505)
+- [ENHANCEMENT] Panel edit mode - show actions popover button when size is too small (#2597)
+- [ENHANCEMENT] `percli dac diff`: add output + avoid early return when processing a directory (#2500)
+- [ENHANCEMENT] DaC CUE SDK: accurate constraints for duration attributes (#2525)
+- [ENHANCEMENT] DaC CUE SDK: add datasource param to the var group builder (#2524)
+- [BUGFIX] Support custom query editors (#2698)
+- [BUGFIX] Discovery: Fix label selector builder and empty value in cuelang (#2723)
+- [BUGFIX] Discovery: Fix the interface impl of the datasource discovery (#2714)
+- [BUGFIX] ListVariable: do not clear input value when selecting a value (#2672)
+- [BUGFIX] Refresh RBAC cache when adding or removing users. (#2670)
+- [BUGFIX] Dashboard copy: hide ephemeral option if disabled (#2633)
+- [BUGFIX] Dashboard edit mode - Expanded panel - delete & drag (#2625)
+- [BUGFIX] CUE lib: Add missing model files (#2612)
+- [BUGFIX] Uses pkg filepath instead of path when manipulating file access (#2611)
+- [BUGFIX] Wait until plugins are loaded to avoid invalid default queries for explore components (#2607)
+- [BUGFIX] Add id to avoid setState loops (#2600)
+- [BUGFIX] Inject API prefix in all UI assets (#2576)
+- [BUGFIX] Login from percli (method not allowed) (#2577)
+- [BUGFIX] ListVariable: fix autocomplete when 'All' is selected (#2554)
+- [BUGFIX] Update unit's migration (#2519)
+- [BUGFIX] GaugeChartPanel: fix empty state (#2548)
+- [BUGFIX] Fixes tooltip display in EDGE / IE (#2547)
+- [BUGFIX] Redirect to the desired page after login success (#2354)
+- [BUGFIX] DaC CUE SDK: Fix inconsistencies in mandatory vs optional attributes in dashboard lib (#2540)
+- [BUGFIX] `percli dac setup`: fix useless requirement of go CLI when using CUE (#2544)
+- [BREAKINGCHANGE] Changes the plugin architure to support remote plugins. (#2154) (#2511) (#2526) (#2545) (#2567) (#2578) (#2582) (#2593) (#2598) (#2606) (#2609) (#2613) (#2616) (#2626) (#2637) (#2660) (#2645) (#2690) (#2719)
```

Comment:
```suggestion
- [BREAKINGCHANGE] Changes the plugin architecture to support remote plugins. (#2154) (#2511) (#2526) (#2545) (#2567) (#2578) (#2582) (#2593) (#2598) (#2606) (#2609) (#2613) (#2616) (#2626) (#2637) (#2660) (#2645) (#2690) (#2719)
```

Reply:
I would also move #2695 in this list, WDYT?

Reply:
ah yeah good catch, for both notes